File: VB/WebSite/App_Code/ExportType.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text

Namespace Grid
	<Flags> _
	Public Enum ExportType
		none = 0
		XLS = 1
		PDF = 2
		CSV = 4
		RTF = 8
	End Enum
End Namespace

File: CS/WebSite/App_Code/ExportType.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Grid
{
    [Flags]
    public enum ExportType
    {
        none = 0,
        XLS = 1,
        PDF = 2,
        CSV = 4,
        RTF = 8
    }
}
